upgrade-tests/src/test/java/io/zeebe/test/RollingUpdateTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.test;
+
+import io.zeebe.containers.ZeebeBrokerContainer;
+import io.zeebe.containers.ZeebeContainer;
+import io.zeebe.containers.ZeebePort;
+import io.zeebe.test.util.AutoCloseableRule;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.lifecycle.Startables;
+
+public class RollingUpdateTest {
+  private static final Logger LOG = LoggerFactory.getLogger(ContainerStateRule.class);
+  private static final String OLD_VERSION = "0.23.3";
```

Comment:
```suggestion
  private static final String OLD_VERSION = VersionUtil.getPreviousVersion();
```

Or replace all occurrences of `OLD_VERSION` to `VersionUtil.getPreviousVersion()`

Reply:
Actually I want to have this hard coded, since we should also be able to do upgrade from pre-previous version if we are really backwards compatible. So ideally we should have tests which starts on the last supported and be able to upgrade to the newest.

Reply:
Do you mean skip minor versions (e.g. 0.23 to 0.25) or do you mean skip micro versions (0.23.x to 0.24.x)?

Reply:
I mean if we say we are backwards compatible this is implicit that this is possible.

Reply:
We typically say we're backwards compatible from one version to the other, I've never heard any mention that we also are with older versions. Was this something that was decided a while back?